src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -0,0 +1,137 @@
+import json
+
+import d3a.constants
+from d3a.d3a_core.redis_connections.redis_area_market_communicator import ResettableCommunicator
+from d3a.models.market import validate_authentic_bid_offer_pair
+from d3a.models.market.market_structures import BidOfferMatch, offer_from_JSON_string, \
+    bid_from_JSON_string
+from d3a.models.myco_matcher.base_matcher import BaseMatcher
+
+
+class ExternalMatcher(BaseMatcher):
```

Comment:
To describe this matching strategy, you've always used the `custom` keyword. I think we should uniform the terminology and either use `custom` or `external`, not both.
So I would vote for either changing this class name to `CustomMatcher` (which I'm not so sure about) or changing the `BidOfferMatchAlgoEnum` to `EXTERNAL`. Or a third option, if you have it!